anomalib/models/components/feature_extractors/feature_extractor.py

```diff
@@ -46,7 +46,7 @@ def __init__(self, backbone: str, layers: List[str], pre_trained: bool = True):
             features_only=True,
             exportable=True,
             out_indices=self.idx,
-        )
+        ).eval()
```

Comment:
would we need to set the eval model in `forward` as well? The reason for asking is because `LightningModule` sets the model to `train` during `training_step`, which potentially puts the `FeatureExtractor` to `train` mode as well?

Reply:
When I compare the state_dict of the trained models they are the same but just to be sure I'll set it to eval in forward as well.